mail/api.py

```diff
@@ -26,34 +28,78 @@ def _mailgun_request(cls, request_func, endpoint, params):
             requests.Response: HTTP response
         """
         mailgun_url = '{}/{}'.format(settings.MAILGUN_URL, endpoint)
+        emails_params = {'from': settings.MAILGUN_FROM_EMAIL}
+        emails_params.update(**params)
         return request_func(
             mailgun_url,
             auth=('api', settings.MAILGUN_KEY),
-            data=dict(**{'from': settings.MAILGUN_FROM_EMAIL}, **params)
+            data=dict(**emails_params)
         )
 
     @classmethod
-    def send_bcc(cls, subject, body, recipients=''):
+    def _recipient_override(cls, body, recipients):
         """
-        Sends a text email to a BCC'ed list of recipients. If the
-        MAILGUN_RECIPIENT_OVERRIDE setting is specified, the list of recipients
+        Helper method to override body and recipients of an email.
+        If the MAILGUN_RECIPIENT_OVERRIDE setting is specified, the list of recipients
         will be ignored in favor of the recipients in that setting value.
 
         Args:
+            body (str): Text email body
+            recipients (list): A list of recipient emails
+
+        Returns:
+            tuple
+        """
+        if settings.MAILGUN_RECIPIENT_OVERRIDE:
+            body = '{0}\n\n[overridden recipient]\n{1}'.format(body, '\n'.join(recipients))
+            recipients = [settings.MAILGUN_RECIPIENT_OVERRIDE]
+        return body, recipients
+
+    @classmethod
+    def send_bcc(cls, subject, body, recipients):
+        """
+        Sends a text email to a BCC'ed list of recipients.
+
+        Args:
             subject (str): Email subject
             body (str): Text email body
-            recipients (str): Comma-separated list of recipient emails
+            recipients (list): A list of recipient emails
 
         Returns:
             requests.Response: HTTP response from Mailgun
         """
-        if settings.MAILGUN_RECIPIENT_OVERRIDE:
-            body = '{}\n\n[overridden recipients]\n{}'.format(body, recipients.replace(',', '\n'))
-            recipients = settings.MAILGUN_RECIPIENT_OVERRIDE
+        body, recipients = cls._recipient_override(body, recipients)
         params = dict(
             to=settings.MAILGUN_BCC_TO_EMAIL,
-            bcc=recipients,
+            bcc=','.join(recipients),
             subject=subject,
             text=body
         )
         return cls._mailgun_request(requests.post, 'messages', params)
+
+    @classmethod
+    def send_batch(cls, subject, body, recipients, chunk_size=1000):
```

Comment:
Should `chunk_size` be configurable via environment variable? I'm not sure myself